configstack/module.go

```diff
@@ -83,7 +84,18 @@ func resolveTerraformModule(terragruntConfigPath string, terragruntOptions *opti
 		return nil, err
 	}
 
-	return &TerraformModule{Path: modulePath, Config: *terragruntConfig, TerragruntOptions: opts}, nil
+	if terragruntConfig.Terraform != nil {
+		s := []string{filepath.Dir(terragruntConfigPath), "*.tf"}
+		matches, err := filepath.Glob(strings.Join(s, ""))
+		if err != nil {
+			return nil, err
+		}
+		if terragruntConfig.Terraform.Source != "" || matches != nil {
```

Comment:
Reading the double-negative is a bit hard. Perhaps it would be simpler to invert this check?

```go
if terragruntConfig.Terraform.Source == "" && matches == nil {
  terragruntOptions.Logger.Printf("Module %s does not have an associated terraform config, sipping...", modulePath)
  return nil, nil
}
```